<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Audio Player</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/buttons/aimp.png</normaloff>:/buttons/aimp.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/buttons/plus.png</normaloff>:/buttons/plus.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_play">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/buttons/play.png</normaloff>:/buttons/play.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/buttons/stop.png</normaloff>:/buttons/stop.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/buttons/skip-next.png</normaloff>:/buttons/skip-next.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_previous">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/buttons/skip-previous.png</normaloff>:/buttons/skip-previous.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_volume">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/buttons/MediumVolume.png</normaloff>:/buttons/MediumVolume.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_pause">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/buttons/pause.png</normaloff>:/buttons/pause.png</iconset>
    </property>
   </widget>
   <widget class="QSlider" name="volume">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>5</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="progress">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTableView" name="playlistView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>380</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="filesSize">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="duration">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="currentTrack">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_openPlaylist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>315</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_savePlaylist">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>315</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
